log4j-core/src/main/java/org/apache/logging/log4j/core/util/NameUtil.java

```diff
@@ -16,44 +16,55 @@
  */
 package org.apache.logging.log4j.core.util;
 
-import java.security.MessageDigest;
-
 import org.apache.logging.log4j.util.Strings;
 
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Objects;
+
 /**
  *
  */
 public final class NameUtil {
 
-    private static final int MASK = 0xff;
-
-    private NameUtil() {
-    }
+    private NameUtil() {}
 
     public static String getSubName(final String name) {
-        if (Strings.isEmpty(name)) {
+        if (Strings.isBlank(name)) {
```

Comment:
In theory this is a behavior change unrelated to this issue, might be good to omit it from this pull request?
(Though it is unlikely that this has a big impact)

Reply:
I can buy that, fixed.